dragonfly/implementations/psyllid_provider.py

```diff
@@ -187,7 +189,7 @@ def set_central_frequency(self, channel, cf):
             return False
         except Exception as e:
```

Comment:
I don't think this error handling is particularly consistent with the rest of the code for two reasons:
- we don't usually bother to catch generic exceptions
  - if we have a specific failure mode, catch the specific exception, logger.critical that this is a known case 
- this only raises a logger.error and then re-raises the same generic exception
  - hopefully whatever catches the exception finally will have all the information in the traceback
(sorry, this isn't new, and doesn't prevent the code from working)

Reply:
Is the difference between logger.critical and logger.error that logger.critical are known errors?

Reply:
for now I added a variable self.mode_testing that distinguishes between testing the acquisition mode and general frequency setting.

Reply:
A broad code review was proposed months ago to straighten out our use of different logger settings, but we haven't done it:
https://github.com/project8/dragonfly/issues/104

The goal for stable running is that we get no logger.critical (or logger.error) messages; if we get such a problem we need to stop and intervene, assess what went wrong.  We use logger.critical to notify of problems we've seen before but hope not to see during normal running - the coldhead is warming, or an ethernet service goes unresponsive.  Usually you don't need to catch generic exceptions because they are caught by other parts of the code.

Logger.warning can be ok to receive during stable running, but should be notified.  We use them when we mute pingers, when a SQL snapshot returns empty, or when you attempt to get/set an endpoint that doesn't have that functionality.